README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://travis-ci.org/freight-team/freight.svg?branch=master)](https://travis-ci.org/freight-team/freight)
 
```

Comment:
I think if you remove this line break they'll show on the same line, though they're not quite the same size.